cli/start.go

```diff
@@ -17,7 +19,22 @@ var (
 )
 
 func startRun(cmd *cobra.Command, args []string) {
-	req, err := createRequest(args)
+	group := ""
+	switch len(args) {
+	case 1:
+		group = args[0]
+	case 0:
+		fallthrough
+	default:
+		cmd.Help()
+		os.Exit(1)
+	}
+
+	newRequestConfig := controller.DefaultRequestConfig()
```

Comment:
I was thinking about a client/server future, and everything we define on the Controller would have an equivalent HTTP route. So if we would migrate this code to HTTP calls, we would do 3 network operations, which feels a bit too much. But we probably just have different thoughts here ;)